src/pill/LinkPill.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { PillContainer, PillValue } from "./shared";
+
+const LinkPillContainer = styled(PillContainer)`
+  a {
+    text-decoration: none;
+  }
```

Comment:
I think you could just move this to the `LinkPillLink` styles?

Reply:
Yup.  Previously I was using an `a` tag directly in the `return` statement below, but when I changed it into the `LinkPillLink` styled component I forgot to move this over.  Great catch 👀